### PR TITLE
Simplify tsds getting started

### PR DESCRIPTION
**Pull request by martijnvg**
* By only using index templates.
* Removing the usage and mentioning of index.routing_path index setting.

### CODE REVIEW

File: docs/reference/data-streams/set-up-tsds.asciidoc [hunk 1]
```diff
@@ -156,88 +183,11 @@ PUT _component_template/my-weather-sensor-mappings
...
   },
-  "_meta": {
-    "description": "Mappings for weather sensor data"
-  }
-}
-----
-// TEST[continued]
-
-[discrete]
-[[tsds-create-index-settings-component-template]]
```
Comment (leemthompo) [inline docs/reference/data-streams/set-up-tsds.asciidoc:173]: 🔍 This id is being referenced in another file, so docs CI is failing

**Comment by leemthompo** [comment]
Would be good to get approval from dev team on details here :)

File: docs/reference/data-streams/set-up-tsds.asciidoc [hunk 2]
```diff
@@ -109,34 +107,52 @@ PUT _ilm/policy/my-weather-sensor-lifecycle-policy
...
-be a plain `keyword` field.
+** Optional: A `date` or `date_nanos` mapping for the `@timestamp` field. If you don’t specify a mapping,
+   Elasticsearch maps `@timestamp` as a `date` field with default options.
 
-Optionally, the template can also include mappings for:
+* Define index settings:
 
-* One or more <<time-series-metric,metric fields>>, marked using the
-`time_series_metric` mapping parameter.
+** Set `index.mode` setting set to `time_series`.
```
Comment (lkts) [inline docs/reference/data-streams/set-up-tsds.asciidoc:133]: typo: "set" is used two times

**Review by lkts — APPROVED** [review]
Nice! Easier to follow for sure.

**Comment by martijnvg** [comment]
@elasticmachine run elasticsearch-ci

**Comment by martijnvg** [comment]
run elasticsearch-ci

**Comment by martijnvg** [comment]
@elasticmachine update branch

**Comment by martijnvg** [comment]
@elasticmachine update branch

**Pull request merged**